src/containers/dashboard.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from "react";
+import "../styles/dashboard.css";
+import nobody from "../assets/images/nobody.jpg";
+
+export default class DashBoard extends Component {
+  
+  constructor(props) {
+    super(props);
+   
+  }
+ 
+  render() { 
+    return(
+      <div className="content">
+      
+        <div className="row dashboard-menu">
+          <div className="col-sm-2 dashboard-menu__left">
+            <p>SoundIT Africa</p>
+            <div className="mobile-search">
+              <form className="mobile-form">
+                <input className="mobile-input" type="search" name="search" placeholder="Search ..."/>
+                <i class="fas fa-search mobile-search-icon"></i>
+              </form>
+            </div>
+          </div>
+          <div className="col-sm-10 dashboard-menu__right">
+          <div className="menu-icon">
+            <input type="checkbox" id="drawer-toggle" name="drawer-toggle"/>
+          <label for="drawer-toggle" id="drawer-toggle-label"></label>
+          <nav id="drawer">
+            <ul className="drawer-menu">
+              <li><a href="#">Vote</a></li>
+              <li><a href="#">Comment</a></li>
+              <li><a href="#">Social</a></li>
+              <li><a href="#">Connect</a></li>
+              <li><a href="#">Music</a></li>
+              <li><a href="#">Videos</a></li>
+              <li><a href="#">Most Liked</a></li>
+              <li><a href="#">Most Viewed</a></li>
+              <li><a href="#">Winners</a></li>
+            </ul>
+        </nav>
+        </div>
+            <div className="right-b">
+              <i class="far fa-envelope right-b__icon"></i>
```

Comment:
should be className as well as the others below,

Reply:
ok